app/resources/forms.js.template

```diff
@@ -359,30 +359,24 @@ function validate_fields() {
 function make_form_expandable(expandedSection) {
   var expandedPart = jQuery(expandedSection).find("div.expanded-part");
   var expandButton = jQuery(expandedSection).find("a.expand-button");
+
   expandedPart.hide();
   expandButton.click(function() {
-    if (expandedPart.is(":visible"))
+    if (expandedPart.is(":visible")) {
       expandedPart.slideUp("slow");
-    else
+      if (expandButton.hasClass('expand-button-icon-up')){
```

Comment:
Yes, I don't need this if-statement
Thank you for pointing it out.
already remove it!